src/@next/SideSheet/SideSheetStyle.ts

```diff
@@ -0,0 +1,86 @@
+import styled from 'styled-components';
+import { Neutral } from '../utilities/colors';
```

Comment:
Incorrect color used for text in header and body. #000 should be replaced with Neutral-B18